### PR TITLE
[DRFT-487] - Adding system-baseline as depedency for Clowder

### PR DESCRIPTION
**Pull request by jpramos123**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [x] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: run_app.sh [hunk 1]
```diff
@@ -1,25 +1,40 @@
-TEMPDIR=`mktemp -d`
+# VALIDATES IF WE HAVE CLOWDER ENV 
+if [ -z "$ACG_CONFIG"]; then
```
Comment (fellipeh) [inline run_app.sh:2]: NIT: Personally, I like to have the "true" condition  on the if

**Review by petrblaho — APPROVED** [review]
LGTM!

**Review by fellipeh — APPROVED** [review]
Nice job here!  LGTM,  I just put a NIT (comment bellow), but it's up to you.. it's really good for me as it is.

File: run_app.sh (hunk 1 above)
Comment (jpramos123) [reply run_app.sh]: I will keep this in mind! I don't make so much changes for now so we can move with the tests!!

Thanks for the advice

**Pull request merged**